shell-ui/babel.config.js

```diff
@@ -1,16 +1,36 @@
-if (process.env.NODE_ENV === "test") {
+if (process.env.NODE_ENV === 'test') {
   module.exports = {
-    "presets": ["@babel/preset-env", "@babel/preset-flow", ["@babel/preset-react", {
-        "runtime": "automatic"
-      }]],
-      "plugins": ["@babel/plugin-transform-modules-commonjs"]
-  }
+    presets: [
+      '@babel/preset-env',
+      '@babel/preset-flow',
+      [
+        '@babel/preset-react',
+        {
+          runtime: 'automatic',
+        },
+      ],
+    ],
+    plugins: ['@babel/plugin-transform-modules-commonjs'],
+  };
 } else {
   module.exports = {
-    "presets": ["@babel/preset-env", "@babel/preset-flow", ["@babel/preset-react", {
-        "runtime": "automatic"
-      }]]
-  }
-  
+    plugins: [
+      [
+        'babel-plugin-styled-components',
+        {
+          namespace: 'shell-ui',
+        },
+      ],
```

Comment:
Do we have a ticket mentioning this behaviour? Could be useful to share with other teams, especially if it creates issues when including the navbar.

Reply:
I just created #3263 to explain the issue and referred it in the commit.